Entwicklung/Cindy/HUSST_Ergebnisdaten_Cindy.xsd

```diff
@@ -1245,42 +1421,150 @@
-			<enumeration value="Abgangskontrolle"/>
-			<enumeration value="Sonstiges"/>
+			<enumeration value="husst.Standardpruefung"/>
+			<enumeration value="husst.Abgangskontrolle"/>
+			<enumeration value="husst.Sonstiges"/>
 		</restriction>
 	</simpleType>
 
-	<simpleType name="Verkehrsmittel_Type">
-		<restriction base="string">
-			<enumeration value="nicht spezifiziert/unbestimmt"/>
-			<enumeration value="Linienbus im Stadtverkehr"/>
-			<enumeration value="Metro/U-Bahn/S-Bahn"/>
-			<enumeration value="Straßenbahn/TRAM"/>
-			<enumeration value="Schiff"/>
-			<enumeration value="ICE"/>
-			<enumeration value="Überlandbus/Regionalbus"/>
-			<enumeration value="Regionalzug"/>
-			<enumeration value="IC"/>
-			<enumeration value="Standseilbahn"/>
-			<enumeration value="Regionalexpress"/>
-			<enumeration value="Expressbus"/>
-			<enumeration value="Flughafenzubringer"/>
-			<enumeration value="Verbundnahverkehr"/>
-			<enumeration value="Verbundnahverkehr ohne Kurzstrecke"/>
-			<enumeration value="SPNV"/>
-			<enumeration value="Verbundnahverkehr ohne Sonderverkehrsmittel"/>
-			<enumeration value="Fähre"/>
-			<enumeration value="Bergbahn"/>
+	<complexType name="TransportserviceArt_Type">
+		<simpleContent>
+			<extension base="husst:TransportserviceArtSimple_Type">
+				<attribute name="Bezeichnung" type="string" use="optional">
+					<annotation>
+						<documentation>
+							Die Angabe der Bezeichnung ist optional und dient nur der Leserlichkeit
+						</documentation>
+					</annotation>
+				</attribute>
+			</extension>
+		</simpleContent>
+	</complexType>
+
+	<simpleType name="TransportserviceArtSimple_Type">
+		<union memberTypes="husst:TransportserviceArtKa_Type husst:ProjektspezifischAnstattKa_Type"/>
+	</simpleType>
+
+	<simpleType name="TransportserviceArtKa_Type">
+		<annotation>
+			<documentation>
+        Die TransportserviceArt kommt aus der VDV-KA.
+        Die KA-Werte kommen aus dem EN1545: TransportTypeCode
+			</documentation>
+		</annotation>
+		<restriction base="husst:INT1">
+			<enumeration value="0">
+				<annotation>
+					<documentation>nicht spezifiziert/unbestimmt</documentation>
+				</annotation>
+			</enumeration>
+			<enumeration value="1">
+				<annotation>
+					<documentation>Linienbus im Stadtverkehr</documentation>
+				</annotation>
+			</enumeration>
+			<enumeration value="3">
+				<annotation>
+					<documentation>Metro/U-Bahn/S-Bahn</documentation>
+				</annotation>
+			</enumeration>
+			<enumeration value="4">
+				<annotation>
+					<documentation>Strassenbahn/TRAM</documentation>
+				</annotation>
+			</enumeration>
+			<enumeration value="6">
+				<annotation>
+					<documentation>Schiff</documentation>
+				</annotation>
+			</enumeration>
+			<enumeration value="10">
+				<annotation>
+					<documentation>ICE</documentation>
+				</annotation>
+			</enumeration>
+			<enumeration value="11">
+				<annotation>
+					<documentation>Ueberlandbus/Regionalbus</documentation>
```

Comment:
In Documentation (hier Überlandbus) auch Umlaute ersetzen? Wenn ja, dann einheitlich auch bei "Fähre" unten